crates/viewer/re_view_spatial/src/view_3d.rs

```diff
@@ -505,7 +505,11 @@ fn view_property_ui_grid3d(
                         .changed()
                         {
                             let color = re_types::components::Color::from(edit_color);
-                            property.save_blueprint_component(ctx, &[color]);
+                            property.save_blueprint_component(
+                                ctx,
+                                &LineGrid3D::descriptor_color(),
+                                &[color],
```

Comment:
nit: this was prexisting, but can't we do this?
```suggestion
                                &color,
```